python/lsst/phosim/utils/phosim_repackager.py

```diff
@@ -199,3 +201,118 @@ def mef_filename(phosim_amp_file, out_dir='.'):
         if outfile.endswith('.gz'):
             outfile = outfile[:-len('.gz')]
         return outfile
+
+    def process_visit_eimg(self, visit_dir, out_dir=None, prefix='lsst',
+                           verbose=False):
+        """Process the visit eimage data.
+
+        Parameters
+        ----------
+        visit_dir: str
+            Directory containing the phosim eimage for a given visit.
+        out_dir : str, optional
+            Output directory for repackaged files. If None, then a directory
+            with name 'eimage' will be created in the cwd. (the default is
+            None.)
+        prefix : str, optional
+            Prefix to use to find PhoSim eimage files in visit_dir. (the
+            default is 'lsst'.)
+        verbose : bool, optional
+            Set to True to print out time for processing each sensor. (the
+            default is False.)
+        """
+
+        phosim_eimg_files \
+            = sorted(glob.glob(os.path.join(visit_dir, f'{prefix}_e_*')))
+
+        if (out_dir is None):
+            out_dir = 'eimage'
+
+        if not os.path.isdir(out_dir):
+            os.mkdir(out_dir)
+
+        for eimg_file in phosim_eimg_files:
+            if verbose:
+                sys.stdout.write(eimg_file + '  ')
+            t0 = time.time()
+
+            # Skip already-processed and compressed files.
+            gzip_file = self.mef_filename_eimg(eimg_file, out_dir=out_dir) + '.gz'
+            if os.path.isfile(gzip_file):
+                sys.stdout.write('%s exists. Skip the processing.' % gzip_file)
+                continue
+
+            print("repackaging", gzip_file)
+            self.repackage_eimg(eimg_file, out_dir=out_dir)
+            if (verbose):
+                print(time.time() - t0)
+                sys.stdout.flush()
+
+    def repackage_eimg(self, phosim_eimg_file, out_dir='.'):
+        """Repackage the phosim eimage file to the format required by obs_lsst.
+
+        Parameters
+        ----------
+        phosim_eimg_file : str
+            PhoSim eimage file.
+        out_dir : str, optional
+            Output directory (the default is '.'.)
+        """
+
+        # Extract the data for each segment from the FITS files
+        # into a dictionary keyed by channel id.
```

Comment:
This comment doesn't seem to line up with this code.

Reply:
Corrects in the new commit: edbbf54.